### PR TITLE
BPM Status message 

### PR DESCRIPTION
**Pull request by glanvl**
Saul did most of the implementation of the status message in ingest. We decided if they were running bricks they would see a ingest message from both ingest wrangler and bricks wrangler but didn't feel this was an issue. Other work will be done around exactly what the users see.
 - Status message in both wranglers
 - Error message in wranglers and methods

### CODE REVIEW

File: ingest_brick_type_method.py [hunk 1]
```diff
@@ -34,6 +35,10 @@ def lambda_handler(event, context):
     logger = general_functions.get_logger()
     # Define run_id outside of try block
     run_id = 0
+
+    # Set status message variables in case it fails before assignment.
+    bpm_queue_url = None
```
Comment (Piwington) [inline ingest_brick_type_method.py:40]: This seems unneeded to me. It should never fail to be assigned as Marshmallow will complain if this variable isn't passed in. This is why no other variables other than run_id are preset like this.

File: ingest_brick_type_wrangler.py [hunk 2]
```diff
@@ -58,6 +60,10 @@ def lambda_handler(event, context):
     error_message = ""
     logger = general_functions.get_logger()
 
+    # Set status message variables in case it fails before assignment.
+    bpm_queue_url = None
+    current_step_num = "1"
```
Comment (Piwington) [inline ingest_brick_type_wrangler.py:65]: What is this variable for? It just looks like it is being passed through directly?

Would prefer if this variable was set alongside all the other lower down to keep them together.
As for URL same as above.

**Review by phunter33 — DISMISSED** [review]
Improved runtime vars for error messages.

File: ingest_takeon_data_method.py [hunk 3]
```diff
@@ -35,6 +36,10 @@ def lambda_handler(event, context):
     logger = general_functions.get_logger()
     # Define run_id outside of try block.
     run_id = 0
+
+    # Set status message variables in case it fails before assignment.
+    bpm_queue_url = None
```
Comment (Piwington) [inline ingest_takeon_data_method.py:41]: Same as above.

File: ingest_takeon_data_wrangler.py [hunk 4]
```diff
@@ -60,6 +62,10 @@ def lambda_handler(event, context):
     error_message = ""
     logger = general_functions.get_logger()
 
+    # Set status message variables in case it fails before assignment.
+    bpm_queue_url = None
+    current_step_num = "1"
```
Comment (Piwington) [inline ingest_takeon_data_wrangler.py:67]: As above

**Review by Piwington — CHANGES_REQUESTED** [review]

File: ingest_brick_type_method.py (hunk 1 above)
Comment (glanvl) [reply ingest_brick_type_method.py:40]: Fair play.

**Review by Piwington — APPROVED** [review]

**Review by Piwington — APPROVED** [review]
Happy for now. Mentioned separately about current_step_num variable to Luke. As seems like we are reintroducing Checkpoint Numbers we deliberately removed. Seems to be there just to get this temporary queue message running so won't have a big debate over it at this time.

**Review by kingmushroom — APPROVED** [review]

**Pull request merged**